### ISSUE TITLE
merlin_tutorials requirements cause conflicts

### ISSUE THREAD

**Issue by DarthAmk97** [RecList/evalRS-CIKM-2022#6]
tensorflow>=2.8
merlin-core>=0.2.0
merlin-models
nvtabular==1.2.2
numpy==1.22

changing requirements.txt to this made it work

For context's sake, this is my pip freeze:
absl-py==1.2.0
anyio==3.5.0
appdirs==1.4.4
argon2-cffi==21.1.0
astunparse==1.6.3
attrs==21.2.0
Babel==2.9.1
backcall==0.2.0
beautifulsoup4==4.10.0
betterproto==1.2.5
bleach==4.1.0
boto3==1.24.34
botocore==1.27.51
bs4==0.0.1
cachetools==4.2.4
certifi==2021.10.8
cffi==1.15.0
charset-normalizer==2.0.7
click==8.1.3
cloudpickle==2.1.0
colorama==0.4.4
cycler==0.11.0
Cython==0.29.21
dask==2022.8.0
debugpy==1.4.1
decorator==5.0.9
deep-translator==1.7.0
defusedxml==0.7.1
dill==0.3.5.1
distributed==2022.8.0
entrypoints==0.3
fasttext==0.9.2
flatbuffers==1.12
fonttools==4.28.1
fsspec==2022.7.1
gast==0.3.3
gensim==4.0.1
google-auth==1.35.0
google-auth-oauthlib==0.4.6
google-pasta==0.2.0
googleapis-common-protos==1.56.4
grpcio==1.41.1
grpclib==0.4.3
h2==4.1.0
h5py==2.10.0
haversine==2.5.1
HeapDict==1.0.1
hpack==4.0.0
htmlmin==0.1.12
hyperframe==6.0.1
idna==3.3
ImageHash==4.2.1
importlib-metadata==4.12.0
importlib-resources==5.4.0
ipykernel==6.0.3
ipython==7.25.0
ipython-genutils==0.2.0
ipywidgets==7.6.5
jedi==0.18.0
Jinja2==3.0.3
jmespath==1.0.1
joblib==1.0.1
json5==0.9.6
jsonschema==4.2.1
jupyter==1.0.0
jupyter-client==6.1.12
jupyter-console==6.4.0
jupyter-core==4.7.1
jupyter-server==1.13.5
jupyterlab==3.2.9
jupyterlab-pygments==0.1.2
jupyterlab-server==2.10.3
jupyterlab-widgets==1.0.2
keras==2.9.0
Keras-Preprocessing==1.1.2
kiwisolver==1.3.2
libclang==14.0.6
llvmlite==0.39.0
locket==1.0.0
Markdown==3.3.4
MarkupSafe==2.0.1
matplotlib==3.4.3
matplotlib-inline==0.1.2
merlin-core==0.5.0
merlin-models==0.6.0
missingno==0.5.0
mistune==0.8.4
msgpack==1.0.4
multidict==6.0.2
multimethod==1.7
multiprocess==0.70.13
nbclassic==0.3.5
nbclient==0.5.8
nbconvert==6.3.0
nbformat==5.1.3
nest-asyncio==1.5.1
networkx==2.6.3
notebook==6.4.12
numba==0.56.0
numpy==1.22.0
nvtabular==1.2.2
oauthlib==3.1.1
opencv-python==4.5.3.56
opt-einsum==3.3.0
ortools==9.2.9972
packaging==21.2
pandas==1.3.5
pandas-profiling==3.1.0
pandocfilters==1.5.0
parso==0.8.2
partd==1.3.0
pathos==0.2.8
phik==0.12.0
pickleshare==0.7.5
Pillow==8.3.1
Pillow-PIL==0.1.dev0
pox==0.3.1
ppft==1.7.6.5
prometheus-client==0.12.0
prompt-toolkit==3.0.19
protobuf==3.19.1
psutil==5.9.1
pyarrow==9.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pybind11==2.10.0
pycparser==2.21
pydantic==1.9.0
Pygments==2.9.0
pyparsing==2.4.7
pyrsistent==0.18.0
python-dateutil==2.8.2
python-dotenv==0.20.0
python-Levenshtein==0.12.2
pytz==2021.3
PyWavelets==1.2.0
pywin32==301
pywinpty==1.1.6
PyYAML==6.0
pyzmq==22.1.0
qtconsole==5.2.0
QtPy==1.11.2
reclist @ git+https://github.com/jacopotagliabue/reclist@4f77db6da6ea8a1a1e828535af3ead15e81cda10
requests==2.26.0
requests-oauthlib==1.3.0
rsa==4.7.2
s3transfer==0.6.0
scikit-learn==1.0.2
scipy==1.8.0
seaborn==0.11.2
selenium==3.141.0
Send2Trash==1.8.0
setuptools-scm==6.3.2
simplejson==3.17.6
six==1.16.0
sklearn==0.0
smart-open==6.0.0
sniffio==1.2.0
sortedcontainers==2.4.0
soupsieve==2.2.1
stringcase==1.2.0
tangled-up-in-unicode==0.1.0
tblib==1.7.0
tensorboard==2.9.1
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.0
tensorflow==2.9.1
tensorflow-estimator==2.9.0
tensorflow-io-gcs-filesystem==0.26.0
tensorflow-metadata==1.9.0
termcolor==1.1.0
terminado==0.12.1
testpath==0.5.0
threadpoolctl==3.1.0
tomli==1.2.2
toolz==0.12.0
tornado==6.1
tqdm==4.62.3
traitlets==5.0.5
typing-extensions==4.0.1
urllib3==1.26.7
visions==0.7.4
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.2.3
Werkzeug==2.0.2
wget==3.2
widgetsnbextension==3.5.2
wrapt==1.13.3
zict==2.2.0
zipp==3.6.0

**Comment by vinid** [comment]
@gabrielspmoreira would you be able to take a look? :)

**Comment by gabrielspmoreira** [comment]
Hi @DarthAmk97 . Have you created a new virtualenv or conda for the competition or you are reusing one with other libraries installed? Which Python version are you using?
What you say is that [notebooks/merlin_tutorial/requirements.txt](https://github.com/RecList/evalRS-CIKM-2022/tree/main/notebooks/merlin_tutorial/requirements.txt) conflicts with the [competition requirements.txt](https://github.com/RecList/evalRS-CIKM-2022/blob/main/requirements.txt) or with other libraries you have installed?

**Comment by Leavingseason** [comment]
thanks @DarthAmk97   I downgraded my TF version to your version: tensorflow==2.9.1 and it fixed my error.
